global.json

```diff
@@ -1,7 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.0",
-    "rollForward": "latestFeature",
-    "allowPrerelease": false
+    "version": "8.0.100",
```

Comment:
Why is this necessary? We have seen issue if we do not use 8.0.0, even that the version officially is 8.0.100

Reply:
What issue specifically? If any, it's probably related to pre-release versions, as 8.0.0 is lower than e.g. 8.0.100-rc.2... But if we want to ensure you're using the final version of the .NET 8 SDK, we should really set this to 8.0.100.

Reply:
I just remember it as built issues..That was the reason for this: https://github.com/umbraco/Umbraco-CMS/pull/15303

When using 8.0.0 and rollForward it seems to work for everyone.
This is also what Rider's UI generates when selecting .NET 8

Reply:
Could the build issues be caused by only having a preview/RC version of .NET 8 SDK installed? Using version 8.0.0 + lastestFeature will satisfy prerelease versions like `8.0.100-rc.2.23502.2` (although you might also need to enable `allowPrerelease` in that case). Version 8.0.100 is the first stable .NET 8 SDK version that was released, so why not explicitly use that as the minimum version?

Currently, version 8.0.101 has been released that contains a security patch, so maybe we should require everyone to update to avoid being vulnerable (similar to how we upgrade transient NuGet dependencies)?